docs/reference/faq.md

```diff
@@ -7,7 +7,68 @@ aren't obviously answered by documentation (often because they span multiple top
 This is the home for such questions! If you have a question not answered here,
 feel free to send us a support request via [support.fossa.com](https://support.fossa.com).
 
-## Does Broker understand FOSSA CLI config files checked into the repository being scanned?
+## General questions
+
+### Where is the data root for Broker?
+
+- On Linux and macOS: `~/.config/fossa/broker/`
+- On Windows: `%USERPROFILE%\.config\fossa\broker`
+
+Elsewhere in these docs we refer to this as though it is an environment variable, e.g. `$DATA_ROOT` or `%DATA_ROOT%`.
+Note that this is not a true environment variable, we just use it this way inside file paths in the documentation
+to make relative file paths clear.
+
+Most Broker subcommands allow customizing the data root via the `-r` flag.
+For more information on this and other runtime customization, run `broker -h`.
+
+### Where is the config file stored?
+
+- On macOS and Linux, the config is stored at `$DATA_ROOT/config.yml`.
+- On Windows, the config is stored at `%DATA_ROOT%\config.yml`.
+
+Most Broker subcommands allow customizing the config location independent of the data root via the `-c` flag.
+For more information on this and other runtime customization, run `broker -h`.
+
+### Where is the local database stored?
+
+- On macOS and Linux, the database is stored at `$DATA_ROOT/db.sqlite`.
+- On Windows, the database is stored at `%DATA_ROOT%\db.sqlite`.
+
+The database may consist of multiple files with the `db.sqlite` prefix, e.g. `db.sqlite-shm` or `db.sqlite-wal`.
+These files are an implementation detail of how Broker accesses the database. 
+If you are moving or deleting the database, ensure that these files are moved or deleted as well.
+
+The database may be deleted at any time so long as Broker is not currently running.
+Note that doing so may cause references in an integration that have already been scanned and uploaded to FOSSA
+to be scanned and uploaded again.
+
+Most Broker subcommands allow customizing the database location independent of the data root via the `-d` flag.
+For more information on this and other runtime customization, run `broker -h`.
+
+### Where is the local task queue stored?
+
+- On macOS and Linux, the queue is stored at `$DATA_ROOT/broker-queue/{queue name}`.
+- On Windows, the queue is stored at `%DATA_ROOT%\broker-queue\{queue name}`.
+
+The local task queue can be deleted at any time so long as Broker is not currently running.
+Note that doing so may cause rework: if Broker has already scanned an integration but not yet uploaded it, 
+and the task queue has been deleted, Broker will need to rescan that integration before it can upload it.
+
+### Where are debug artifacts stored?
+
+Debug artifacts are stored at the `debugging.location` configured in the [config file](./config.md#debugging).
```

Comment:
Should we mention the default location here?

Reply:
Whoops, I had enabled auto merge, sorry. I think though that since the default location is in the referenced doc it's okay to leave it out, but happy to make a new PR if you think we should add it.